client/src/components/Auth/SignIn.tsx

```diff
@@ -73,6 +79,11 @@ export default class SignIn extends React.Component<ISignInProps> {
                         />
                     </div>
                 </form>
+                {authStoreServerError && (
```

Comment:
The condition and wrapping divs should be inside of Error component
You should call it like this:
`<Error isDisplaying={authStoreServerError } />`